gcloud/exceptions.py

```diff
@@ -147,7 +147,7 @@ class InternalServerError(ServerError):
     code = 500
 
 
-class NotImplemented(ServerError):
+class NotImplementedMethod(ServerError):
     """Exception mapping a '501 Not Implemented' response."""
```

Comment:
ISTM that `MethodNotImplemented` is a clearer mapping of the HTTP response code.